tasks/configure.yml

```diff
@@ -80,7 +80,9 @@
   loop: "{{ barman_check_result.results }}"
   loop_control:
     label: "{{ item.pg_server_name }}"
-  when: item.stdout_lines | select('match', '^' ~ item.pg_server_name ~ '\.replication slot:\ FAILED') | list | length > 0
+  when:
+  - barman_create_slots
```

Comment:
One last thing, let's add `| bool` here so that `"false"` (string) is not treated as `true`, which may happen either by mistake or when the value is provided with `-e barman_create_slots=false` as an command line parameter.